CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ Changes
 * [UI/Developer]: Fixed issue where search for member to add to group resulted in no results found when search term contained capital letters.
 * [Documentation]: Added information on fixing ONLY_FULL_GROUP_BY sql error to the administrator faq docs.
 * [UI]: User on a remote project with a project role of manager has the ability to assign user groups to the project.
+* [UI]: New dedicated cart page.
```

Comment:
Could you add a statement also saying that we've updated the interface for exporting to Galaxy, which now must be done through the cart?

Reply:
Updated in c96767bc71fb37d639ea0713566d5aefb5fcaada

Reply:
Thanks. Could you also say in that statement that we've deprecated exporting to Galaxy through the Project Samples page?

Reply:
Updated in 55ebdd8315e756894c5edd1fd6a8e5d34ea84099